index.html

```diff
@@ -1,210 +1,272 @@
 
-    <div class="white-screen"></div>
+        <p>html</p>
 
-    <h2>Multi-Post Stories</h2>
+        <div class="vertical-line"></div>
 
-    <p class="work-p">
-      A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
-    </p>
+        <p>bootstrap</p>
 
-    <div class="vertical-lines flex">
-      <p>css</p>
+        <div class="vertical-line"></div>
 
-      <div class="vertical-line"></div>
+        <p>Ruby</p>
+      </div>
 
-      <p>html</p>
+      <button type="button">See Project</button>
 
-      <div class="vertical-line"></div>
+      <div class="white-screen"></div>
 
-      <p>bootstrap</p>
+      <h2>Multi-Post Stories</h2>
 
-      <div class="vertical-line"></div>
+      <p class="work-p">
+        A daily selection of privately personalized reads; no accounts or
+        sign-ups required. has been the industry's standard dummy text ever
+        since the 1500s, when an unknown printer took a standard dummy text.
+      </p>
 
-      <p>Ruby</p>
-    </div>
+      <div class="vertical-lines flex">
+        <p>css</p>
 
-    <button type="button">See Project</button>
+        <div class="vertical-line"></div>
 
-    <div class="white-screen"></div>
+        <p>html</p>
 
-    <h2>Multi-Post Stories</h2>
+        <div class="vertical-line"></div>
 
-    <p class="work-p">
-      A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
-    </p>
+        <p>bootstrap</p>
 
-    <div class="vertical-lines flex">
-      <p>css</p>
+        <div class="vertical-line"></div>
 
-      <div class="vertical-line"></div>
+        <p>Ruby</p>
+      </div>
 
-      <p>html</p>
+      <button type="button">See Project</button>
+    </main>
 
-      <div class="vertical-line"></div>
+    <footer>
+      <section>
+        <h2>CONTACT US</h2>
 
-      <p>bootstrap</p>
+        <p class="footer-p">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </section>
 
-      <div class="vertical-line"></div>
+      <form class="flex" action="https://formspree.io/f/mayajwdo" method="POST">
 
-      <p>Ruby</p>
-    </div>
+        <fieldset>
+          <label for="name"></label>
+          <input type="text" name="name" id="name" placeholder="Text input" minlength="5" maxlength="30" required/>
+  
+          <label for="email"></label>
+          <input type="email" name="email" id="email" placeholder="Email address" required/>
 
-    <button type="button">See Project</button>
+          <textarea name="user-message" placeholder="Enter text here" minlength="3" maxlength="500" required></textarea>
```

Comment:
I suggest message input's type equals textarea as stated in the project requirements

Reply:
@Davidosky007 Sry can you be more explicit, I don't understand. do you mean give textarea a label an put the same name?

Reply:
` <textarea name="user-message" type="textarea" placeholder="Enter text here" minlength="3" maxlength="500" required></textarea>`   (something like this )

Reply:
I run npx hint . but give a error with that changes.  
![Screen Shot 2021-06-08 at 20 32 11](https://user-images.githubusercontent.com/67211919/121278709-ccb61000-c898-11eb-9c33-258bb441a6ae.png)

Reply:
ok you can remove it 🙏 continue working on the other changes 👍